lib/ardb/adapter/base.rb

```diff
@@ -20,6 +20,25 @@ def foreign_key_drop_sql(*args); raise NotImplementedError; end
   def create_db(*args); raise NotImplementedError; end
   def drop_db(*args);   raise NotImplementedError; end
 
+  def migrate_db
+    verbose = !(ENV["MIGRATE_VERBOSE"] == "false")
```

Comment:
@Criptak let's switch this to `_QUIET` and just check that it has a non-nil value set.